cmd/connections_register_android.go

```diff
@@ -0,0 +1,29 @@
+// +build android
+
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package cmd
+
+import (
+	"github.com/mysteriumnetwork/node/core/node"
+)
+
+func (di *Dependencies) registerConnections(nodeOptions node.Options) {
+	di.registerOpenvpnConnection(nodeOptions)
```

Comment:
This can be removed too. We redefine openvpn connection for android anyway

Reply:
And only noop will be left here. :D
Should I remove both?

Reply:
No. Noop should work on all oses (and it kind of works :D)

Reply:
removed openvpn.